src/components/WatchList/WatchList.tsx

```diff
@@ -0,0 +1,87 @@
+import TickerRow from "../TickerRow";
+
+const initialWatchList = ["S:VTI", "S:SPY", "C:BTC-USD"];
+
+type WatchListProps = {
+  wlLoading: boolean;
```

Comment:
Just an idea: one thing I've seen done before when you don't want to pass a whole bunch of data up and down the component stack is to use [graphql-batch-loader](https://www.apollographql.com/docs/react/api/link/apollo-link-batch-http/) which allows you to break up a huge graphql query and place the relevant pieces inside the components that need them, and then the link will batch them all up. Not saying we should do that now, just pointing it out as an option if this pattern becomes annoying to manage.